README.md

```diff
@@ -5,42 +5,65 @@
 <img src="https://travis-ci.org/bow-swift/nef.svg?branch=develop">
 </a>
 <a href="https://github.com/bow-swift/nef">
-<img src="https://img.shields.io/badge/platform-iOS%20%7C%20macOS-orange.svg" alt="Platforms">
+<img src="https://img.shields.io/badge/platform-macOS-orange" alt="Platforms">
+</a>
+<a href="https://github.com/bow-swift/nef">
+<img src="https://img.shields.io/badge/dependency%20manager-CocoaPods%20%7C%20Carthage-orange" alt="CocoaPods+Carthage compatible">
 </a>
 <a href="https://gitter.im/bowswift/bow">
 <img src="https://img.shields.io/badge/gitter-nef-blueviolet.svg" alt="Gitter">
 </a>
 
 </p>
 
-`nef`, short for [Nefertiti](https://en.wikipedia.org/wiki/Nefertiti), mother of Ankhesenamun, is a command line tool to ease the creation of documentation in the form of Xcode Playgrounds. It provides compile-time verification of documentation, exports it in Markdown format that can be consumed by [Jekyll](https://jekyllrb.com/) to generate websites, and export [Carbon](https://carbon.now.sh/) snippets for given Xcode Playground.
+`nef`, short for [Nefertiti](https://en.wikipedia.org/wiki/Nefertiti), mother of Ankhesenamun, is a set tool to ease the **creation of documentation** in the form of Xcode Playgrounds. It provides compile-time **verification** of documentation, exports it in **Markdown** format that can be consumed by [Jekyll](https://jekyllrb.com/) to generate websites, and export [Carbon](https://carbon.now.sh/) snippets for given Xcode Playground.
 
 `nef` is inspired by [ΛNK](https://github.com/arrow-kt/ank) for Kotlin and [tut](https://github.com/tpolecat/tut) for Scala.
 
-## 📥 Installation
+### Features
+
+💡 Eases the creation of Xcode Playgrounds with support for [__third party libraries__](#-creating-a-xcode-playground).
+
+💡 [__Compiles Xcode Playgrounds__](#-compiling-a-xcode-playground) with support for third party libraries from the command line.
 
-`nef` can be installed using [Homebrew](https://brew.sh). `nef` needs Xcode and [Cocoapods](https://cocoapods.org) as dependencies. It will warn you if there is a missing dependency and will provide guidance to install it.
+💡 Generates [__Markdown__](#-generating-a-markdown-project) project from Xcode Playground.
+
+💡 Generates Markdown files that can be consumed from [__Jekyll__](#-generating-markdown-files-for-jekyll) to create a microsite.
+
+💡 Export [__Carbon__](#-exporting-carbon-code-snippets) code snippets for given Xcode Playgrounds.
+
+&nbsp;
+
+## 💻 Installation
+
+📟 **Using** [**Homebrew**](https://github.com/bow-swift/homebrew-nef) (preferred)
```

Comment:
Why not using `###` syntax to mark sub sections under `Installation`?

Reply:
Not sure why I do not use it; changed! Thanks